### PR TITLE
Removes some incorrect capitalization where I saw them

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
# Document the changes in your pull request

names of objs aren't supposed to be capitalized because they look weird when examining them.
Ex: ``They are wearing a Head of Security's Belt``.

There's a lot of wrong varedits that do this and it's a shame, but this is mostly just what I found in code.

# Changelog

:cl:  
spellcheck: Removed a lot of incorrectly capitalized names in things.
/:cl:

### CODE REVIEW

File: code/game/objects/items/holy_armours.dm [hunk 1]
```diff
@@ -205,15 +205,15 @@
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/darktemplar
 
 /obj/item/clothing/suit/space/hardsuit/darktemplar/chap
-	name = "Black Templar chaplain battle armor"
+	name = "black Ttmplar chaplain battle armor"
```
Comment (Marmio64) [inline code/game/objects/items/holy_armours.dm]: accidental spelling error addition

**Review by Aquizit — APPROVED** [review]
The very limited map stuff in this is fine.

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
proper names and abbreviations should at least stay capitalized

**Review by Aquizit — DISMISSED** [review]
I mean yeah there's no real map shit going on here it's just A -> a

**Pull request closed without merging**